09. cli-and-http-request/clihelper/RunAsync.go

```diff
@@ -0,0 +1,16 @@
+package clihelper
+
+import "log"
+
+func RunAsync(c CliBindingProperties) stdInParameter {
+	var stdout, stderr []byte
+	var stdInParameter stdInParameter
```

Comment:
`var stdInParameter stdInParameter` to `var stdInParameter StdInParameters`
and add paramter

NewStdInParamters(cmd) -> where it will take these line 9,10 as encapsulation

Reply:
replaced line 9,10 with this >> stdInParameter := NewStdInParameters(c.Cmd)

created another go file NewStdInParameter and defined 
func NewStdInParameters(cmd *Cmd) *StdInParameter {
	StdInParameter.stdoutIn, _ = cmd.StdoutPipe()
	StdInParameter.stderrIn, _ = cmd.StderrPipe()
	return *StdInParameter
}